davenmathews/Bright_Events
# #153206252 Implement Event Deletion

## This pull request implements event deletion

This is on top of other previous commits

**The tasks completed are**
DELETE /api/v1/events/<int:event_id>

**This can be manually tested by:**
1. Clone the repository.
2. Use virtualenv to start a virtual environment.
3. Ensure the users.json and events.json are placed in your virtualenv folder and that they are empty
3. Install the requirements.txt.
4. Run appv1.py and open the local host in your browser.
5. Test the endpoints by registering, logging in and out, creating, modifying, and deleting an event.


**Constrains:**
Cannot access dashboard when user is not logged in
Will not modify event if the user in session is not the owner of the event
Will not delete event if the user in session is not the owner of the event

**It Finishes #153206252 story on PT**
